anta/tests/mlag.py

```diff
@@ -42,21 +40,25 @@ class VerifyMlagStatus(AntaTest):
     @AntaTest.anta_test
     def test(self) -> None:
         """Main test function for VerifyMlagStatus."""
+        self.result.is_success()
         command_output = self.instance_commands[0].json_output
+
+        # Skipping the test if MLAG is disabled
         if command_output["state"] == "disabled":
             self.result.is_skipped("MLAG is disabled")
             return
-        keys_to_verify = ["state", "negStatus", "localIntfStatus", "peerLinkStatus"]
-        verified_output = {key: get_value(command_output, key) for key in keys_to_verify}
-        if (
-            verified_output["state"] == "active"
-            and verified_output["negStatus"] == "connected"
-            and verified_output["localIntfStatus"] == "up"
-            and verified_output["peerLinkStatus"] == "up"
-        ):
-            self.result.is_success()
-        else:
-            self.result.is_failure(f"MLAG status is not OK: {verified_output}")
+
+        # Verifies the localIntfStatus
+        if command_output.get("negStatus") != "connected":
+            self.result.is_failure("MLAG Negotiation status is not connected")
```

Comment:
Can we  use Expected and Actual output here?

```suggestion
            self.result.is_failure(f"MLAG Negotiation status mismatch - Expected: connected. Actual: {command_output.get("negStatus")}")
```
Update the same at other places as well.

Reply:
Updated. Thanks!!